<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1288</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JSON-–ë–ª–æ–∫–Ω–æ—Ç Pro</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="controls_layout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="close_doc_btn">
        <property name="text">
         <string>‚ùå –ó–∞–∫—Ä—ã—Ç—å –¥–æ–∫—É–º–µ–Ω—Ç</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_font">
        <property name="text">
         <string>–®—Ä–∏—Ñ—Ç:</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QFontComboBox" name="font_combo">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_size">
        <property name="text">
         <string>–†–∞–∑–º–µ—Ä:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="font_size">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="minimum">
         <number>6</number>
        </property>
        <property name="maximum">
         <number>72</number>
        </property>
        <property name="value">
         <number>12</number>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignTop">
       <widget class="QPushButton" name="text_color_btn">
        <property name="text">
         <string>üé® –¶–≤–µ—Ç —Ç–µ–∫—Å—Ç–∞</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bg_color_btn">
        <property name="text">
         <string>üñåÔ∏è –¶–≤–µ—Ç —Ñ–æ–Ω–∞</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tab_widget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="editor_tab">
       <attribute name="title">
        <string>üìù –†–µ–¥–∞–∫—Ç–æ—Ä</string>
       </attribute>
       <layout class="QHBoxLayout" name="editor_layout">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QTextEdit" name="text_edit"/>
          <widget class="QTreeWidget" name="tree_widget">
           <column>
            <property name="text">
             <string>–°—Ç—Ä—É–∫—Ç—É—Ä–∞ JSON</string>
            </property>
           </column>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>–ü–∞–Ω–µ–ª—å –∏–Ω—Å—Ç—Ä—É–º–µ–Ω—Ç–æ–≤</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFormat"/>
   <addaction name="actionMinify"/>
   <addaction name="actionValidate"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1288</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>–§–∞–π–ª</string>
    </property>
    <widget class="QMenu" name="menuRecentFiles">
     <property name="title">
      <string>–ù–µ–¥–∞–≤–Ω–∏–µ —Ñ–∞–π–ª—ã</string>
     </property>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="menuRecentFiles"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>–ò–Ω—Å—Ç—Ä—É–º–µ–Ω—Ç—ã</string>
    </property>
    <addaction name="actionFind"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>–°–ø—Ä–∞–≤–∫–∞</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>–û—Ç–∫—Ä—ã—Ç—å</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>–°–æ—Ö—Ä–∞–Ω–∏—Ç—å</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>–°–æ—Ö—Ä–∞–Ω–∏—Ç—å –∫–∞–∫</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>–í—ã—Ö–æ–¥</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>–ù–∞–π—Ç–∏</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>–≠–∫—Å–ø–æ—Ä—Ç</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>–û –ø—Ä–æ–≥—Ä–∞–º–º–µ</string>
   </property>
  </action>
  <action name="actionFormat">
   <property name="text">
    <string>‚ú® –§–æ—Ä–º–∞—Ç–∏—Ä–æ–≤–∞—Ç—å JSON</string>
   </property>
  </action>
  <action name="actionMinify">
   <property name="text">
    <string>üì¶ –ö–æ–º–ø–∞–∫—Ç–Ω—ã–π –≤–∏–¥</string>
   </property>
  </action>
  <action name="actionValidate">
   <property name="text">
    <string>‚úîÔ∏è –ü—Ä–æ–≤–µ—Ä–∏—Ç—å</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
